src/BlazingPizza.Client/Pages/Index.razor

```diff
@@ -72,7 +72,7 @@
         // to sign in first if needed
         if (await UserState.TrySignInAsync())
         {
-            var newOrderId = wait HttpClient.PostJsonAsync<int>("orders", OrderState.Order);
+            var newOrderId = await HttpClient.PostJsonAsync<int>("orders", OrderState.Order);
```

Comment:
This can be `Order` instead of `OrderState.Order`

Reply:
Done. Thank you for the review!